code/__HELPERS/510.dm

```diff
@@ -0,0 +1,253 @@
+// Helpers for running 510 code on older versions of BYOND.
+#if DM_VERSION < 510
+#define BYGEX "code/__HELPERS/bygex.dll"

```

Comment:
this is going to break the linux instructions I have because you're now directing to the exact path instead of the inexact one previously